tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -48,3 +48,15 @@ c
 true
 <function>
 ```
+
+## Nothing and Null
+`Nothing` is a subtype of all types. There is no value that has type `Nothing`. There are two uses: The first is to signal abnormal termination such as an exception (i.e. it doesn't return). The second type usage is an element type of empty collections (e.g. `Set[Nothing]`).
+
```

Comment:
A few suggested edits:

`Nothing` is a subtype of all types, also called the bottom type. There is no value that has type `Nothing`. There are two common uses: the first is to signal non-termination such as a thrown exception, program exit, or an infinite loop (i.e., it is the type of an expression which does not evaluate to a value, or a method that does not return normally); the second common usage is as the element type of empty collections (e.g. `Set[Nothing]`).